egglog-ast/src/generic_ast_helpers.rs

```diff
@@ -357,6 +400,31 @@ where
             }
         }
     }
+
+    pub fn make_unresolved(self) -> GenericAction<String, String> {
```

Comment:
document this, and any other new public methods.

Reply:
also, should these make_unresolved impls be an instance of a generic 'map' method?